### PR TITLE
Create Heapster in cluster

### PR DESCRIPTION
**Pull request by zreigz**
It deploys Heapster in cluster to make it reachable via service proxy:
http://localhost:8080/api/v1/proxy/namespaces/kube-system/services/heapster/

### CODE REVIEW

File: build/cluster.js [hunk 1]
```diff
@@ -121,7 +152,7 @@ gulp.task('wait-for-heapster', function(doneFn) {
 
     // constantly query the heapster until it is properly running
     heapsterHealthCheck(function(result) {
-      if (result.length > 0) {
+      if (result === runningState) {
```
Comment (bryk) [inline build/cluster.js]: Hmm... The result is always "Running". There's no other type. You should get rid of the result variable here.

File: src/test/integration/zerostate/zerostate_test.js [hunk 2]
```diff
@@ -12,15 +12,16 @@
...
+  // let page;
 
   beforeEach(() => {
     browser.get('#/zerostate');
-    page = new PageObject();
+    // page = new PageObject();
   });
 
-  it('should do something', () => { expect(page.deployButton.getText()).toContain('DEPLOY'); });
+  // TODO: Enable this test to test zerostate page.
```
Comment (bryk) [inline src/test/integration/zerostate/zerostate_test.js]: Say why it is disabled. For future developers to know.

File: build/heapster.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Deploys Heapster in cluster.
+
+PATH_ROOT=$(dirname "${BASH_SOURCE}")/
+# Port of the apiserver to serve on.
+PORT=8080
+
```
Comment (bryk) [inline build/heapster.sh]: add `set -e` line here to exit the script in case of any errors.

**Comment by bryk** [comment]
Looks very good. A few style comments. PTAL.

File: build/cluster.js (hunk 1 above)
Comment (zreigz) [inline build/cluster.js]: I fixed heapsterHealthCheck to return correct result not always runningState value

File: src/test/integration/zerostate/zerostate_test.js (hunk 2 above)
Comment (zreigz) [inline src/test/integration/zerostate/zerostate_test.js]: done

File: build/heapster.sh (hunk 3 above)
Comment (zreigz) [inline build/heapster.sh]: done

**Comment by zreigz** [comment]
PTAL

**Comment by bryk** [comment]
LGTM

**Comment by zreigz** [comment]
Thanks for review

**Pull request merged**